marlowe-actus/src/Language/Marlowe/ACTUS/QCGenerator.hs

```diff
@@ -9,7 +9,8 @@ import qualified Data.Map                                          as M
 import           Data.Time
 import           Data.Time.Clock.POSIX                             (posixSecondsToUTCTime)
 import           Language.Marlowe.ACTUS.Analysis                   (genProjectedCashflows)
-import           Language.Marlowe.ACTUS.Definitions.BusinessEvents
+import Language.Marlowe.ACTUS.Definitions.BusinessEvents
+    ( RiskFactors(..) )
```

Comment:
strange formatting, has stylish haskell been run?

Reply:
Nope. I’m fixing my NixOs VM today coz it took all space on my hard drive

Reply:
And then resizing broke it lol